converters/fromZigbee.js

```diff
@@ -4508,22 +4508,29 @@ const converters = {
         cluster: 'aqaraOpple',
         type: ['attributeReport', 'readResponse'],
         convert: (model, msg, publish, options, meta) => {
-            const mappingButton = {
-                1: 'left',
-                2: 'center',
-                3: 'right',
-            };
+            if (!msg.data.hasOwnProperty('512')) {
+                console.log('aqaraOpple missing 512 property.', msg.data);
```

Comment:
Use `meta.logger` instead.

Reply:
Oops. Didn't mean to leave that in, it's a debug logging statement because I'm pretty sure logging the absence of a property there is just log spam under normal usage.